front-end/src/components/main-page/MainPage.jsx

```diff
@@ -131,19 +131,19 @@ class MainPage extends Component {
                       Enter the Project ID that you want to analyze
                     </span>
                   </div>
-                </div>
-              </div>
-              <div className="flex sm:absolute sm:py-52">
+                  <div className="flex sm:absolute mx-auto sm:py-12">
                 <label className="text-l flex font-cabin -mt-7 text-center">
                   <input
-                  className= "appearance-none h-4 w-4 border border-gray-300 rounded-sm bg-off-white checked:bg-purple-300 checked:border-blue-600 focus:outline-none transition duration-200 mt-1 align-top bg-no-repeat bg-center bg-contain float-left mr-2 cursor-pointer"
+                  className= "appearance-none h-4 w-4 border border-gray-300 rounded-sm bg-off-white checked:bg-purple-300 checked:border-blue-600 focus:outline-none focus:border-purple-300 focus:ring-purple-300 transition duration-200 mt-1 align-top bg-no-repeat bg-center bg-contain float-left mr-2 cursor-pointer"
                   type="checkbox" 
                   data-testid="checkbox"
                   onChange={this.toggleMerge}/>
                   (Beta) Merge similar questions
                 </label>
```

Comment:
We are handling this label incorrectly. I'll send you a SS of how to do this nicer.